include/swift/SIL/Dominance.h

```diff
@@ -95,11 +95,10 @@ class DominanceInfo : public DominatorTreeBase {
 /// Postcondition: each block in \p frontier is dominated by \p root and either
 /// exits the function or has a single successor that is not dominated by \p
 /// root.
-///
-/// With no critical edges, the dominance frontier is identified simply by leaf
-/// blocks in the dominance subtree.
-void computeDominanceFrontier(SILBasicBlock *root, DominanceInfo *domTree,
-                              SmallVectorImpl<SILBasicBlock *> &frontier);
+
+void computePredecessorsOfDominanceFrontierDominatedByBlock(
```

Comment:
@atrick Please suggest if there's a technical term for "predecessors of dominance frontier dominated by block" : )